packages/user/src/services/invite.ts

```diff
@@ -109,8 +152,10 @@ export default class InviteService<
 
     return jwt.sign(data, jwtSecret, {
       expiresIn,
+      jwtid: crypto.randomUUID(),
```

Comment:
question: Why was this added?

Reply:
adding a jwtid ensures that the token is actually regenerated instead of the same token being issued for the same data. We could also include the expiry_date to achieve this, but an id like this seemed better

Reply:
Got it, thanks. Makes sense. Feel free to merge as you please :)